pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -63,6 +63,12 @@ type volumemgrContext struct {
 	pubBaseOsPersistStatus  pubsub.Publication
 	subBaseOsPersistStatus  pubsub.Subscription
 
+	subContentTreeResolveStatus pubsub.Subscription
+	pubContentTreeResolveConfig pubsub.Publication
+	subContentTreeConfig        pubsub.Subscription
+	pubContentTreeStatus        pubsub.Publication
+	pubAppAndImageToHash        pubsub.Publication
```

Comment:
Isn't this at least misnamed? You don't know about App UUIDs.

Reply:
Resolved